recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "python-igraph" %}
-{% set version = "0.7.1.post6" %}
+{% set version = "0.7.1.post7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a5ea42790a077eadb3d1e8f82edc47516fea4a2b26bbf52bfc148370fe0fe676
+  url: https://github.com/igraph/python-igraph/archive/release-0.7.tar.gz
```

Comment:
this is a small issue, the auto-tick bot will not be able to create updates for this package since you "pinned" the url, consider using a truncation of the version variable instead.

Reply:
Yes, sorry for the late reply, I indeed know that it will not be updated automatically. The "problem" is that there has not been any new version release, but the particular bugs that are resolved in this `0.7.1.post7` version are quite relevant I believe. So, I think this is OK for the moment. The release cycle seems relatively slow anyhow, so it isn't a problem to manually maintain this (for now anyway).